src/test/java/wbooks/userdata/userdata.feature

```diff
@@ -0,0 +1,42 @@
+Feature: get data from specific user
+
+  Background:
+    * url 'https://wbooks-api-stage.herokuapp.com/api/v1'
+
+    * def login = callonce read('classpath:wbooks/login/login.feature@login')
+    * def authToken = login.access_token
+
+    * def userDataSchema =
+    """
+    {
+      id: '#number',
+      email: '#string',
+      last_name: '#string',
+      first_name: '#string',
+      rents_counter: '#number',
+      comments_counter: '#number',
+      image_url: '##string'
+    }
+    """
+
+  Scenario: get user data
+    * def userId = 2
+    Given path 'users/' + userId
+    And header Authorization = authToken
+    When method get
+    Then status 200
+    And match response == userDataSchema
+    And match response.id == userId
+
+  Scenario: user data not authorized
+    * def fakeToken = 'eyFaKeToKen123'
+    Given path 'users/1'
```

Comment:
use this way to define the path: Given path 'users' userId and define userId to background in order to define all the variables in one section